### DISCUSSION TITLE
menghapus / memfilter background hitam di halaman ahu.go.id

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#1381] [Personal Filter]
https://ahu.go.id/pencarian/profil-pt/?nama=black%20diamonds&tipe=perseroan&page=2&g-recaptcha-response=03AL8dmw_KA5UiSSvHVB4YtPxVSGexDRfHupe0Egm1c7___pFe8iz3tj66useTqefUzw_jOotzCzPVbdMKPptYQobJqm8gtfzmiG96e1H227ouTarDFpq8YOosU0e7kTjQy4Q5vYTBP1BlTDSoGtuoMWtncVu6l7LvS4Jtcusu3V26IsWfwIMcFDkdnnEjQlPENzmfM4qwZpLHJJ0im3kqFz-axvQbKSiR19l_TqJZLGbnKm6DkWuKnqqbDaIe_j6ZUPRM6Viq3QsQJvFI3psyZZ8mnPCbVSugZGE95KsqkRztTCxy3gjN4_xc3fOw7c6SVxdG3L5IZ7HawJmmuW565aneZMS_hbILlE3JRn_HNtuR-O6lQwzG5ykmZu0FXzoMl5qJ3-In2CH2u6r5vt9fQr2MympG3f2H8ZGKOqKuF9UH8linh-LgpKA71GmtzNs1nK9glI0j020KTNanKXTxwDCdbrsg3fwemiAVJwG14cu0qQc0S1tdpaSubrQ5qt2QMueDjss4aIOhWS59s0B2tas4CVVUJ2Rmbj9BGEN8G9pwWX25TOv2NWk

bagaimana cara menghapus background hitam ?
![image](https://user-images.githubusercontent.com/19517680/236668804-08711625-fbf9-4423-848d-332f8a13f89e.png)

**Comment by realodix** [comment]
Mereka menambahkan aturan css di body, jadi tinggal kita masukkan saja CSSnya..

Pada contoh di bawah ini Saya hanya mematikannya saja, agar ia menggunakan background default dari situs tersebut.

```adblock
ahu.go.id##body:style(background: none !important;)
```

![nonameh](https://user-images.githubusercontent.com/1314456/236674279-2e1e765a-12c9-478f-a86c-29610e8f2784.png)

**Reply by 3xploiton3** [reply]
terimakasih @realodix sangat membantu sekali
